backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -134,6 +151,8 @@
     SWAGGER_TAG__USER_ENDPOINTS, SWAGGER_TAG_USER_SUBSCRIPTIONS_SECTION
 )
 
+ALLOWED_MIMETYPE_FOR_AVATAR = ["image/png", "image/jpeg", "image/webp", "image/bmp", "image/gif"]
```

Comment:
This is not fully tested, we should probably verify each type here (if pillow doesn't broke running it) , and check also how the behavior with a faked image.

Reply:
I do tested, i have to update list to support bmp and for a weird reason httpie consider webp as plain text.
This is working for all these type and trying to fake it will give error 500 with error message which make sense.

Reply:
@inkhey did you discuss with @gbonaspetti about supported mime types ?

Why do you limit supported mimetypes? Does it avoid tracim to work if the user send an unknown filetype? I think this behavior is an enhancement and may limit the usage of tracim because we need to *explicitly* define supported image formats (which is for sure less than what PIL supports)

Reply:
@lebouquetin i just followed the specification here : https://github.com/tracim/tracim/issues/3842

Reply:
This is typically what I call over-quality because it is not required for tracim to work to implement this limitations - and it limits users with no reason (except we did not test all image types).

Why not allowing TIFF, for example?

(note: my point is not critical *but* the decision seems symptomatic to me)